src/plugins/performanceIndicator/plugin.js

```diff
@@ -58,5 +65,58 @@ export default function PerformanceIndicator() {
         indicator.statusClass('s-status-error');
       }
     }
+
+    function showOverlay() {
+      const overlayStylesText = `
+          #c-performance-indicator--overlay {
+            background-color:rgba(0,0,0,0.5);
+            position: absolute;
+            width: 300px;
+            left: calc(50% - 300px);
+          }
+      `;
+      const overlayMarkup = `
+        <div id="c-performance-indicator--overlay" aria-label="Performance Overlay">
```

Comment:
aria implies interactivity and I don't think this is an interactive component. In this case, we should default to a raw label

Reply:
I'm not sure sure I understand. You want me to add a new `<label>` element here? I don't want the label to be visible though? I can add a `title` attribute to the existing `<div>`, would that work?

Reply:
Then just use `getByTitle()` in the test?

Reply:
OK, have switched to title, which feels more semantically correct than `aria-label` and is natively supported by playwright. Lemme know if that works.